src/particle/__init__.py

```diff
@@ -38,3 +30,27 @@
 # Direct access to handy LaTeX to HTML particle name conversions
 # Direct access to kinematics functions
 from .particle import latex_to_html_name, lifetime_to_width, width_to_lifetime
+
+__all__ = (
+    "Geant3ID",
+    "Charge",
+    "Inv",
+    "InvalidParticle",
+    "Parity",
+    "Particle",
+    "ParticleNotFound",
+    "PythiaID",
+    "PDGID",
+    "Pythia2PDGIDBiMap",
+    "latex_to_html_name",
+    "lifetime_to_width",
+    "width_to_lifetime",
+    "Status",
+    "SpinType",
+    "__version__",
+)
+
+
+def __dir__():
+    # type: () -> Tuple[str, ...]
+    return __all__
```

Comment:
These should be in most "public" files, and should help tab completion be more precise. Anything not in `__all__` will not show up in tab completion (Python 3.7+ only). @eduardo-rodrigues please check `__all__` and make sure I didn't miss anything, or add something that should not be public. It should be pretty easy, since most things in `__init__.py` files are either in `__all__` or have a `noqa` since they are unused (in which case they are basically hidden).

Reply:
This I did not know. Thank you for the info. OK, will look at that for DecayLanguage - extra on my to-do list.